cli/src/main/kotlin/tools/samt/cli/DiagnosticFormatter.kt

```diff
@@ -28,17 +28,25 @@ class DiagnosticFormatter(
 
     private fun format(): String = buildString {
 
-        // print context-less messages
+        // keep track of the number of errors and warnings
+        var countsBySeverity = mutableMapOf<DiagnosticSeverity, Int>()
```

Comment:
The following line fixes a IntelliJ warning and probably greatly improves performance:
`val countsBySeverity = EnumMap<DiagnosticSeverity, Int>(DiagnosticSeverity::class.java)`

Reply:
`greatly improves performance` lol